Client/FluentClient.cs

```diff
@@ -96,7 +96,7 @@ public virtual IRequest SendAsync(HttpRequestMessage message)
 
             // clone the underlying message because HttpClient doesn't normally allow re-sending
             // the same request, which would break IRequestCoordinator.
-            return new Request(message, this.Formatters, request => this.BaseClient.SendAsync(request.Message.Clone(), request.CancellationToken), this.Filters.ToArray())
+            return new Request(message, this.Formatters, request => this.BaseClient.SendAsync(request.Message.Clone(), request.CancellationToken), this.Filters)
```

Comment:
You're right, the request filters is now a reference to client filters. I'll fix. I removed `ToArray` because collection becausomes fixed in size am we can't add/remove items. The alternative is to use `ToList`.